### PR TITLE
Product v8 migration

### PR DESCRIPTION
**Pull request by guewen**
Migrate products!

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
Amazing work, thanks! :+1:

File: addons/product/migrations/8.0.1.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+               #
+
+               ]
+    for sql in queries:
+        execute(cr, sql)
+
+
+def copy_fields(cr, pool):
+    product_tmpl= pool['product.template']
+    # copy the active field from product to template
```
Comment (StefanRijnhart) [inline addons/product/migrations/8.0.1.1/post-migration.py:58]: Oh, flake8 warns about the missing space here in line 57

**Comment by guewen** [comment]
Yep, missed this one. Should be better now.

File: addons/mrp/migrations/8.0.1.1/post-migrate.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from openerp.openupgrade import openupgrade
```
Comment (bwrsandman) [inline addons/mrp/migrations/8.0.1.1/post-migrate.py:1]: This file should be called `post-migration.py`
Also, there are pep8 issues in this script.

File: addons/stock_account/migrations/8.0.1.1/post-migrate.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
```
Comment (bwrsandman) [inline addons/stock_account/migrations/8.0.1.1/post-migrate.py:1]: File should be called `post-migration.py`

**Comment by bwrsandman** [comment]
A few files have the wrong name. Have they been running?

**Comment by StefanRijnhart** [comment]
Everything called [pre|post]-*.py will run. Non-blocking IMHO, we can clean up and rename according to some convention later on.

**Comment by bwrsandman** [comment]
In that case, the .travis.yml file needs to reflect this. 
This PR has pep8 issues which weren't caught.

**Comment by StefanRijnhart** [comment]
All is well on pep-bot after changing the file pattern :+1:

File: addons/mrp/migrations/8.0.1.1/post-migrate.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+from openerp import pooler, SUPERUSER_ID
+
+def move_fields(cr, pool):
```
Comment (bwrsandman) [inline addons/mrp/migrations/8.0.1.1/post-migrate.py:4]: This function is incomplete:
- not closed bracket
- unused variables

**Comment by guewen** [comment]
@komsas the CI actually returned a green result because it was not considering a file in mrp, that's why I didn't caught the issues. This is the only file having issues, I will make a new PR (note that mrp still needs to  be migrated).

**Pull request merged**